components/scream/src/mct_coupling/CMakeLists.txt

```diff
@@ -19,13 +19,13 @@ if ("${SCREAM_DYNAMICS_DYCORE}" STREQUAL "HOMME")
   # We set them to a default, but each compset should set its values anyways.
   set (SCREAM_DYN_TARGET none CACHE STRING "The name of the desired Homme target.")
   set (SCREAM_NP 4 CACHE STRING "The number of Gauss points per element.")
-  set (SCREAM_NUM_LEV FALSE CACHE STRING "The number of vertical levels.")
+  set (SCREAM_NUM_VERTICAL_LEV FALSE CACHE STRING "The number of vertical levels.")
```

Comment:
I don't think you need this. All the other SCREAM_blah variables in this file are not variables that standalone scream already uses (they are meant to be fed from cime). On the other hand, `SCREAM_NUM_VERTICAL_LEV` is already declared in the scream's main CMakeLists file, so this line has no effect (and also misleading, since it has a default value of FALSE for a string).

I'd simply purge this line.